client/src/App.js

```diff
@@ -33,16 +33,16 @@ function App() {
             <Route path="/profile">
               <Profile />
             </Route>
+            <Route path="/login">
+              <LogIn />
+            </Route>
+            <Route path="/googleSignUp">
+              <SignUpGoogle email={email} />
+            </Route>
+            <Route path='/' >
+              <SignUpEmail handleEmailEntry={handleEmailEntry} />
+            </Route>
           </UserProvider>
```

Comment:
I have these routes wrapped in the UserProvider context component which is allowing me to access the googleSignUp route to login, but the res.redirect from the back-end auth/google/callback is rendering the Profile page on-top of the login.